mobile/src/bundles/users/components/image-picker/image-picker.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { PermissionsAndroid } from 'react-native';
+import { type StyleProp, type ViewStyle } from 'react-native';
+import { launchCamera, launchImageLibrary } from 'react-native-image-picker';
+import { type ImagePickerResponse } from 'react-native-image-picker';
+
+import { Button, Modal } from '~/bundles/common/components/components';
+import { ButtonType } from '~/bundles/common/enums/enums';
+import { useCallback, useState } from '~/bundles/common/hooks/hooks';
+
+type ImagePickerProperties = {
+    label: string;
+    onImageLoad: (payload: Promise<ImagePickerResponse>) => void;
+    containerStyle?: StyleProp<ViewStyle>;
+};
+
+const ImagePicker: React.FC<ImagePickerProperties> = ({
+    label,
+    onImageLoad,
+    containerStyle,
+}) => {
+    const [isPopUpActive, setIsPopUpActive] = useState(false);
+
+    const getImageFromLibrary = useCallback((): void => {
+        setIsPopUpActive(false);
+        const result = launchImageLibrary({ mediaType: 'photo' });
+        onImageLoad(result);
+    }, [onImageLoad]);
+
+    const getImageFromCamera = useCallback((): void => {
+        PermissionsAndroid.request(PermissionsAndroid.PERMISSIONS.CAMERA, {
+            title: 'App Camera Permission',
+            message: 'App needs access to your camera ',
+            buttonNeutral: 'Ask Me Later',
+            buttonNegative: 'Cancel',
+            buttonPositive: 'OK',
+        })
+            .then((grantedCamera) => {
+                if (grantedCamera === PermissionsAndroid.RESULTS.GRANTED) {
+                    const result = launchCamera({
+                        mediaType: 'photo',
+                        saveToPhotos: true,
+                    });
+                    onImageLoad(result);
+                }
+                setIsPopUpActive(false);
+            })
+            .catch(() => {
+                /*TODO: Notification error*/
+            });
+    }, [onImageLoad]);
+
+    const onPickerPress = useCallback(() => {
+        setIsPopUpActive(true);
+    }, []);
+
+    const onPopUpClose = useCallback(() => {
+        setIsPopUpActive(false);
+    }, []);
+
+    return (
+        <>
+            <Modal visible={isPopUpActive} onClose={onPopUpClose}>
+                <Button label="Camera" onPress={getImageFromCamera} />
+                <Button label="Library" onPress={getImageFromLibrary} />
+            </Modal>
```

Comment:
Let's modify it a bit, for ex. 
<img width="321" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/103518438/8127992c-e95c-4241-8f3f-8ac3dae21411">
Let's make buttons outlined or lust with text splitted by dividers

Reply:
fix